ReFix/Modules/Repairs/RepairsListView.swift

```diff
@@ -67,15 +32,45 @@ struct RelpairsListView: View {
 }
 
 extension RelpairsListView {
-    var details: some View {
-        HStack {
-            Text("Деталі")
-                .font(.system(size: 17))
-                .font(.system(.title3))
-                .foregroundStyle(.secondary)
+    
+    func listRepairRow(repair: Binding<Repair>) -> some View {
+        ZStack {
+            HStack {
+                VStack(alignment: .listRowSeparatorLeading, content: {
+                    Text("\(repair.wrappedValue.brand) \(repair.wrappedValue.model)")
+                        .font(.system(size: 17))
+                        .padding(1)
+                    Text("#\(repair.id)")
+                        .font(.system(size: 15))
+                        .foregroundStyle(.secondary)
+                })
+                Spacer()
+                NavigationLink(destination: {
+                    RepairDetailView()
+                        .environmentObject(RepairDetailViewModel(repair: repair))
+                }, label: {})
+                .frame(maxWidth: 30)
+            }
+            .padding(.horizontal)
             
-            Image(systemName: "chevron.right")
-                .opacity(0.5)
+            RoundedRectangle(cornerRadius: 10, style: .continuous)
+                .stroke(Color(uiColor: .tertiaryLabel), lineWidth: 1)
+        }
+        .listRowSeparator(.hidden)
```

Comment:
better to use list setups in main body (line 17)

Reply:
it is not working

Reply:
<img width="598" alt="Screenshot 2024-03-02 at 10 25 41" src="https://github.com/vanyalaptii/ReFix/assets/51946521/c7151f61-f4ba-4a5a-adb7-1e91c84eb9df">